hack/upload-tutorial-resources.sh

```diff
@@ -20,23 +20,48 @@ then
     exit
 fi
 
+# this array consists of <oci-path of blueprint>;<local path of blueprint>;<default version tag>
+# the version tag can be set inline in the blueprint.yaml file itself by the comment line
+# # TUTORIAL_BLUEPRINT_VERSION: <my version>
+# this is the default that we fall back on in case the version information is not maintained
+# in the blueprint file
 blueprints=(
-  "eu.gcr.io/gardener-project/landscaper/tutorials/blueprints/ingress-nginx:${BLUEPRINT_INGRESS_NGINX_VERSION},./docs/tutorials/resources/ingress-nginx/blueprint"
-  "eu.gcr.io/gardener-project/landscaper/tutorials/blueprints/echo-server:${BLUEPRINT_ECHO_SERVER_VERSION},./docs/tutorials/resources/echo-server/blueprint"
-  "eu.gcr.io/gardener-project/landscaper/tutorials/blueprints/simple-aggregated:${BLUEPRINT_AGGREGATED_VERSION},./docs/tutorials/resources/aggregated/blueprint"
+  "eu.gcr.io/gardener-project/landscaper/tutorials/blueprints/ingress-nginx;./docs/tutorials/resources/ingress-nginx/blueprint;v0.2.1"
+  "eu.gcr.io/gardener-project/landscaper/tutorials/blueprints/echo-server;./docs/tutorials/resources/echo-server/blueprint;v0.1.1"
+  "eu.gcr.io/gardener-project/landscaper/tutorials/blueprints/simple-aggregated;./docs/tutorials/resources/aggregated/blueprint;v0.1.1"
 )
 component_descriptors=(
   "./docs/tutorials/resources/ingress-nginx"
   "./docs/tutorials/resources/echo-server"
   "./docs/tutorials/resources/aggregated"
+  "./docs/tutorials/resources/local-ingress-nginx"
```

Comment:
I think the external jsonschema tutorial is missing?

Reply:
*grml* Forgot that one... fixed.